static/js/iframe-common.js

```diff
@@ -91,8 +94,12 @@ export function generateIFrame(domain, runtimeConfig) {
   iFrameResize({
     checkOrigin: false,
     onInit: function() {
+      iframeInitialized = true;
+      iframeMessageQueue.forEach(message => {
+        sendToIframe(message);
+      })
       runtimeConfig && sendToIframe({
```

Comment:
If there's a `runtimeConfig`, could we added it do the `iframeMessageQueue`? Then we wouldn't need this special case here, we could use the message queue loop above.

Reply:
sure, I like that approach